wikimedia/wikipedia-ios
# Fix nav bar in place on modal

**Fixes Phabricator ticket:** https://phabricator.wikimedia.org/T266898

### Notes
* Merging this into the new navigation branch

### Test Steps
1. Run on iPad.
1. Open an AaaLD modal.
1. Tap a "View discussion" button. (Article on United States has one fairly close to the top.)
1. Ensure the nav bar doesn't disappear when you scroll down.
